js/plates-view/plate-mesh.ts

```diff
@@ -352,9 +351,10 @@ export default class PlateMesh {
     } else if (colormap === "age") {
       this.material.uniforms.usePatterns.value = false;
       this.material.uniforms.colormap.value = this.colormapTextures.age;
-      // Fields at the divergent boundary should have 0 age.
-      this.defaultColormapValue = 0;
-      this.defaultColorAttr = { r: 0, g: 0, b: 0, a: 0 };
+      // Fields at the divergent boundary should have max age to avoid strange color transition around preexisting
+      // crust borders.
+      this.defaultColormapValue = 1;
+      this.defaultColorAttr = PREEXISTING_CRUST_COLOR;
```

Comment:
These two lines are actually very important. You can try to set `defaultColormapValue` to `0` and `defaultColorAttr` to any other color to see the difference.

This might become important if Amy or Chris complain about this small artifact:
<img width="738" alt="Screenshot 2022-05-06 at 20 05 03" src="https://user-images.githubusercontent.com/767857/167235700-60ad4ac0-2388-4efb-8e70-b994c1c0b8ce.png">

However, I don't see a better solution for now. If we set defaultColormapValue to 0, then each plate will have yellow border which doesn't seem to make much sense.

Reply:
I don't suppose there's some simple smoothing operation we could perform when rendering?

Reply:
The problem here is that what you see between the plates is a special type of a field - `adjacentField`. It's a field that is not yet part of the model, but it's a "field candidate". It's added to the model when it can be (there's enough of gap between plates). I'm rendering these fields so there are no visible gaps between plates, plus they have a few more functions. And the mentioned lines affect their rendering properties as far as I remember.